packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -333,8 +333,8 @@ ShareActivity.create = function create(conversation, object, spark) {
     parent: spark
   });
 
-  if (files) {
-    files.forEach((file) => share.add(file));
+  if (files && files.items) {
+    files.items.forEach((file) => share.add(file));
```

Comment:
the problem here is this change breaks the interface to makeShare(). It should probably be something like 
```js
const files = files.items || files
files.forEach(...)
```